bundles/org.openhab.binding.boschshc/src/main/java/org/openhab/binding/boschshc/internal/devices/camera/CameraHandler.java

```diff
@@ -0,0 +1,167 @@
+import static org.openhab.binding.boschshc.internal.devices.BoschSHCBindingConstants.CHANNEL_PRIVACY_MODE;
+
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeoutException;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.openhab.binding.boschshc.internal.devices.BoschSHCHandler;
+import org.openhab.binding.boschshc.internal.exceptions.BoschSHCException;
+import org.openhab.binding.boschshc.internal.services.cameranotification.CameraNotificationService;
+import org.openhab.binding.boschshc.internal.services.cameranotification.CameraNotificationState;
+import org.openhab.binding.boschshc.internal.services.cameranotification.dto.CameraNotificationServiceState;
+import org.openhab.binding.boschshc.internal.services.privacymode.PrivacyModeService;
+import org.openhab.binding.boschshc.internal.services.privacymode.PrivacyModeState;
+import org.openhab.binding.boschshc.internal.services.privacymode.dto.PrivacyModeServiceState;
+import org.openhab.core.library.types.OnOffType;
+import org.openhab.core.thing.ChannelUID;
+import org.openhab.core.thing.Thing;
+import org.openhab.core.types.Command;
+
+/**
+ * Handler for security cameras.
+ * <p>
+ * This implementation handles services and commands that are common to all cameras, which are currently:
+ * 
+ * <ul>
+ * <li><code>PrivacyMode</code> - Controls whether the camera records images</li>
+ * <li><code>CameraNotification</code> - Enables or disables notifications for the camera</li>
+ * </ul>
+ * 
+ * <p>
+ * The Eyes outdoor camera advertises a <code>CameraLight</code> service, which unfortunately does not work properly.
+ * Valid states are <code>ON</code> and <code>OFF</code>.
+ * One of my two cameras returns <code>HTTP 204 (No Content)</code> when requesting the state.
+ * Once Bosch supports this service properly, a new subclass may be introduced for the Eyes outdoor camera.
+ * 
+ * @author David Pace - Initial contribution
+ *
+ */
+@NonNullByDefault
+public class CameraHandler extends BoschSHCHandler {
+
+    private PrivacyModeService privacyModeService;
+    private CameraNotificationService cameraNotificationService;
+
+    public CameraHandler(Thing thing) {
+        super(thing);
+        this.privacyModeService = new PrivacyModeService();
+        this.cameraNotificationService = new CameraNotificationService();
+    }
+
+    @Override
+    protected void initializeServices() throws BoschSHCException {
+        super.initializeServices();
+
+        this.registerService(this.privacyModeService, this::updateChannels, List.of(CHANNEL_PRIVACY_MODE));
+        this.registerService(this.cameraNotificationService, this::updateChannels,
+                List.of(CHANNEL_CAMERA_NOTIFICATION));
+    }
+
+    @Override
+    public void initialize() {
+        super.initialize();
+        requestInitialStates();
```

Comment:
The long poll results do not include the initial camera states. Is there a more elegent way to obtain the initial states?

Reply:
If this becomes more common, we could think about moving the code from your `requestInitial...` methods into the `registerService` method, e.g. with a flag that the initial state should be actively fetched. But if the long poll does not contain the information I do not see a way to get the state otherwise.

Reply:
ok, then I will leave it as it is for now and it can be generalized later if necessary.